samples-lib/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <style name="OutlineButton" parent="@style/Widget.MaterialComponents.Button.OutlinedButton">
+        <item name="android:textColor">@color/outlinebutton_textcolor</item>
+        <item name="android:backgroundTint">@color/outlinebutton_background</item>
+    </style>
+
+    <style name="RegularButton" parent="@style/Widget.MaterialComponents.Button.TextButton">
+        <item name="android:textColor">@color/textbutton_textcolor</item>
+        <item name="android:backgroundTint">@color/textbutton_background</item>
+        <item name="iconTint">@color/textbutton_textcolor</item>
+    </style>
+
+    <style name="TextInputLayoutTheme" parent="@style/Widget.MaterialComponents.TextInputLayout.FilledBox.ExposedDropdownMenu">
+        <item name="boxStrokeColor">#00FFFFFF</item>
+    </style>
```

Comment:
Could you help me understand the reason for this custom theme? Seems like it is only used in one sample, and I find it helpful to have a stroke color to know which view is currently in focus. 

Would prefer if this could be removed.

Reply:
I havn't added it. Just moved this theme from `add-features-with-contingent-values/styles.xml` to `samples-lib/styles.xml` since we don't have the styles for individual samples anymore.

Reply:
I believe you worked on this sample? `add-features-with-contingent-values`
do you want me to remove it? its being used in `add-feature-layout.xml` file to set the style for `TextInputLayout`

Reply:
Remove it 👍

Reply:
Removed the custom style and added `style="@style/Widget.MaterialComponents.TextInputLayout.FilledBox.ExposedDropdownMenu"` to the textInputLayout's in respective xml of the sample.